tests/unit/environments/BinaryLabelData.h

```diff
@@ -0,0 +1,86 @@
+#ifndef BINARY_LABEL_DATA_HPP
+#define BINARY_LABEL_DATA_HPP
+
+#ifdef HAVE_LAPACK
```

Comment:
Because the generate_gaussians in DataGenerator class require LAPACK. http://shogun.ml/api/latest/DataGenerator_8cpp_source.html